packages/shared/src/components/sources/SourceCard.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { ReactElement, ReactNode } from 'react';
+import classNames from 'classnames';
+import { useRouter } from 'next/router';
+import SquadMemberShortList from '../squads/SquadMemberShortList';
+import { Card } from '../cards/Card';
+import { Button } from '../buttons/Button';
+import { Connection } from '../../graphql/common';
+import {
+  SourceMember,
+  SourceMemberRole,
+  SourceType,
+} from '../../graphql/sources';
+import { Image } from '../image/Image';
+import { SquadDirectoryCardBannerDefault } from '../../lib/constants';
+import { SquadJoinButton } from '../squads/SquadJoinButton';
+import { Origin } from '../../lib/analytics';
+
+interface SourceCardProps {
```

Comment:
These are quite a lot of props, would it not be better to accept `source` as most of these belong to a source?